### PR TITLE
Gw 92: Added sponsorship partner contact info

### PR DESCRIPTION
**Pull request by msamanthaf**
Does this align with what you had in mind? Any suggestions/anything to add?
Deploy preview: https://deploy-preview-139--gisaubcv3.netlify.app/

image

### CODE REVIEW

File: client/src/pages/partners/components/PartnerContact.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+				Interested to partner with us?
+			</Typography>
+			<Box className='flex-col lg:flex-row mt-10'>
+				<Box className='flex lg:flex-col'>
+					<Typography
+						variant='button'
+						color='primary'
+						className='border-4 border-primary rounded-full px-3 sm:px-7 sm:py-2'
+					>
+						<a href='mailto:contact.gisau@gmail.com'>Contact us at sponsorship.gisau@gmail.com</a>
```
Comment (chrishadrian) [inline client/src/pages/partners/components/PartnerContact.jsx:26]: a should be wrapping Box imi

**Review by chrishadrian — COMMENTED** [review]
Looks good so far! I don’t know about the button tho? Should we ask our designers to design it for us? I’d also want to redesign how to access our sponsors not gonna lie.

**Review by chrishadrian — CHANGES_REQUESTED** [review]
Can you remove halloween page from the website too?

**Comment by msamanthaf** [comment]
Dy think it's better to keep the font the way it was before or the reduced/smaller version?

**Comment by chrishadrian** [comment]
Font looks good but spacing still needs to change:
image. I'd say make it to have an even space

**Comment by chrishadrian** [comment]
And don't forget to remove halloween event from current event 👍

**Review by chrishadrian — APPROVED** [review]
LOOKS GOODDD!!

**Pull request merged**